.ci/azure-pipelines/docs-pipeline.yaml

```diff
@@ -0,0 +1,46 @@
+trigger:
+  paths:
+    include:
+    - doc
+
+pr:
+  paths:
+    include:
+    - doc
+
+resources:
+  pipelines:
+    - pipeline: Build-CI
+      source: Build
+      trigger:
+        stages:
+        - build_gcc
+  containers:
+    - container: fmt # for formatting.yaml
+      image: pointcloudlibrary/fmt
+    - container: doc # for documentation.yaml
+      image: pointcloudlibrary/doc
+    - container: env1804 # for tutorials.yaml
+      image: pointcloudlibrary/env:18.04
+
+stages:
+  - stage: formatting
+    displayName: Formatting
+    # if docs pipeline triggered by build_gcc stage,
+    # the formatting stage has already run, thus it
+    # won't run for a second time here.
+    condition: not(eq(variables['Build.Reason'], 'ResourceTrigger'))
+    jobs:
+      - template: formatting.yaml
+
+  - stage: documentation
+    displayName: Documentation
+    condition: in(dependencies.formatting.result, 'Succeeded', 'SucceededWithIssues', 'Skipped')
```

Comment:
`ne` here?
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#ne

Reply:
Oh yes, L38 can be changed to `ne(in(dependencies.formatting.result, 'Failed', 'Canceled'))`, Thank you! :grinning:

Reply:
Ah, my bad. Not there. I meant `ne` to replace the `not(eq(`, not here. 🤦🏾

Reply:
Not here, but for L32

Reply:
I'll push the change immediately. Thanks for reviewing!